### PR TITLE
Fix k6 script

### PR DESCRIPTION
**Pull request by aomegax**
Created a bash script `postman-to-k6-converter.sh` to generate automatically the js script for k6 tool.
The js script is created by postman-to-k6 tool and then manipulated in order to adapt the generated request for file upload.

Edited the deploy pipeline to use the previous script for k6 generation code.

#### List of Changes

- edit deploy pipeline
- edit run_k6_local.sh
- edit postman environment json
- add postman-to-k6-converter.sh
- add formData-k6-dep.js

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: api-test/run_k6_local.sh [hunk 1]
```diff
@@ -2,12 +2,31 @@
...
 
 if [[ "$(pwd)" =~ .*"api-test".* ]]; then
     cd ..
 fi
+
+# convert postman collection into k6 script
 postman-to-k6  api-test/ApiConfig.postman_collection.json -e api-test/local.postman_environment.json -o ./k6-script.js
-k6 run --http-debug=full --vus 2 --duration 5s ./k6-script.js
+
+# modify generated script to solve file upload problem
```
Comment (pasqualespica) [inline api-test/run_k6_local.sh:16]: Could you enhance comment description ? To explain the solved problem and all these things done with `awk`, not really immediate to read at first sight

**Review by pasqualespica — COMMENTED** [review]
great : ) , bui i saw other lot stuffs that i think isn't part of the fix k6 script.

Can you verify with rebase ?

File: api-test/run_k6_local.sh (hunk 1 above)
Comment (aomegax) [reply api-test/run_k6_local.sh]: done

**Review by jacopocarlini — CHANGES_REQUESTED** [review]
can you do the same fix in the pipeline script too?

**Review by jacopocarlini — APPROVED** [review]
🚀

**Pull request closed without merging**